app/src/test/java/edu/uc/reedws/musiclink/PlaylistLocalDatabaseTest.kt

```diff
@@ -1,17 +1,6 @@
 package edu.uc.reedws.musiclink
 
-import org.junit.Assert.*
-
-import androidx.arch.core.executor.testing.InstantTaskExecutorRule
-import androidx.lifecycle.MutableLiveData
-import edu.uc.reedws.musiclink.dto.PlaylistDTO
-import edu.uc.reedws.musiclink.service.PlaylistService
-import edu.uc.reedws.musiclink.ui.main.MainViewModel
-import io.mockk.every
-import io.mockk.mockk
-import org.junit.Rule
 import org.junit.Test
-import org.junit.rules.TestRule
```

Comment:
Removed the unused imports to clear the clutter from the codebase. You might use it in the future, but I didn't see any use of these imports right now.

Reply:
Control-Alt-O